target/ppc/fpu_helper.c

```diff
@@ -3368,84 +3368,199 @@ static inline target_ulong compute_fp_flags(float_status *excp_ptr)
 
 EXCP_CHECK(float32)
 EXCP_CHECK(float64)
-
 #undef EXCP_CHECK
+
 #define GET_VsrBF(A,I) A->VsrBF(I)
 #define GET_VsrHF(A,I) A->VsrHF(I)
 #define GET_VsrSF(A,I) A->VsrSF(I)
 #define GET_VsrDF(A,I) A->VsrDF(I)
+/* Defines that will never be called as they're in !FLOAT code */
+#define GET_VsrSBF(A,I) 0
+#define GET_VsrSHF(A,I) 0
+#define GET_VsrSSF(A,I) 0
+#define GET_VsrSDF(A,I) 0
```

Comment:
If the macros are left separated, then a GET_Vsr* shouldn't be necessary in floating point version as it could be changed for the direct Vsr call.